lsp-pyls.el

```diff
@@ -269,6 +269,12 @@ dot."
   :group 'lsp-pyls
   :package-version '(lsp-mode . "6.1"))
 
+(defcustom lsp-pyls-plugins-autopep8-enabled t
+  "Enable or disable the plugin."
+  :type 'boolean
+  :group 'lsp-pyls
+  :package-version '(lsp-mode . "6.2"))
+
 (defcustom lsp-pyls-plugins-yapf-enabled t
```

Comment:
Please disable the yapf by default here as well,

Thank you

Reply:
Done!